cmd/manager/main.go

```diff
@@ -36,6 +36,7 @@ func main() {
 	machineActuator := machine.NewActuator(machine.ActuatorParams{
 		MachineClient: cs.MachineV1beta1(),
 		CoreClient:    mgr.GetClient(),
+		EventRecorder: mgr.GetRecorder("gcp-controller"),
```

Comment:
After merging https://github.com/openshift/cluster-api-provider-gcp/pull/26 the code needs to be changed:

`s/mgr.GetRecorder("gcp-controller")/mgr.GetEventRecorderFor("gcpcontroller")`